src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerReferenceDataClient.java

```diff
@@ -133,26 +166,35 @@ private HttpHeaders getMultipleAuthHeaders(String role) {
     private Map getResponse(ResponseEntity<Map> responseEntity) {
 
         Map response = objectMapper
-            .convertValue(
-                responseEntity.getBody(),
-                Map.class);
+                .convertValue(
+                        responseEntity.getBody(),
+                        Map.class);
 
         response.put("http_status", responseEntity.getStatusCode().toString());
         response.put("headers", responseEntity.getHeaders().toString());
         response.put("body", responseEntity.getBody());
         return response;
     }
 
+    private Map getDeleteResponse(ResponseEntity<Map> responseEntity) {
+
+        Map<String, Object> response = new HashMap<>();
+
+        response.put("status", responseEntity.getStatusCode().toString());
+        response.put("headers", responseEntity.getHeaders().toString());
+        return response;
+    }
+
     private final String getBearerToken(String userId, String role) {
         return generateToken(issuer, expiration, userId, role);
     }
 
     public static String generateS2SToken(String serviceName) {
         return Jwts.builder()
-            .setSubject(serviceName)
-            .setIssuedAt(new Date())
-            .signWith(SignatureAlgorithm.HS256, TextCodec.BASE64.encode("AA"))
-            .compact();
+                .setSubject(serviceName)
```

Comment:
I think let's move this method in common class Utils. I did that but it may reused accross

Reply:
I did not actually create this method, all I have done is format the class...

Reply:
Yes I know I created this method :) if  possible refactor in util hence we can use across

Reply:
Ah okay! Sure I will refactor 😄

Reply:
I think it is already in util

Reply:
@abhijitdiwan  - to echo what @SujeethKumarAri has said, this method is currently in the Util class for Integration Tests